### PR TITLE
create copyright

### PR DESCRIPTION
**Pull request by testing-abc17777**

### CODE REVIEW

**Review by testing-abc17777 — COMMENTED** [review]
Succes create footer with dynamic date

File: src/components/foot.vue
Comment (oktaaa17) [inline src/components/foot.vue:1]: done

File: src/main.js
Comment (oktaaa17) [inline src/main.js:1]: done

**Review by testing-abc17777 — COMMENTED** [review]
done

**Review by testing-abc17777 — COMMENTED** [review]
done

**Comment by testing-abc17777** [comment]
done

**Comment by testing-abc17777** [comment]
add footer

File: src/App.vue [hunk 1]
```diff
@@ -17,4 +17,5 @@ onMounted(async () => {
 <template>
     <navbarComponent />
     <mainComponent />
+    <footer-vue />
```
Comment (oktaaa17) [inline src/App.vue:20]: nice

File: src/components/foot.vue [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<script setup>
+import { ref } from 'vue'
+const currentYear = ref(new Date().getFullYear())
```
Comment (oktaaa17) [inline src/components/foot.vue:4]: nice

File: src/main.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ import aboutImage from '@/components/aboutImage.vue'
 import filterSearch from '@/components/filterSearch.vue'
 import filterDropdown from '@/components/filterDropdown.vue'
 import filterHeight from '@/components/filterHeight.vue'
+import footerVue from '@/components/foot.vue'
```
Comment (oktaaa17) [inline src/main.js:16]: nice

**Review by testing-abc17777 — COMMENTED** [review]
done create footer with dyanamic date

**Pull request merged**